src/config.json

```diff
@@ -20,7 +20,7 @@
         "extension_url": "https://s3.amazonaws.com/extend.brackets/{0}/{0}-{1}.zip",
```

Comment:
Revert this file change.  It's something that Brackets does via grunt, but we don't want (again, for merging easily).